mozilla_django_oidc_db/signals.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.auth.signals import user_logged_in
+from django.dispatch import receiver
+
+
+@receiver([user_logged_in], dispatch_uid="oidc.set_config_class")
```

Comment:
```suggestion
@receiver([user_logged_in], dispatch_uid="oidcdb.set_config_class")
```
let's not pollute the upstream library namespace and make it clear this is "our thing". Settings etc. should also follow this approach - everything we define/use that is not provided by the upstream library needs the `OIDCDB_` prefix :)